### ISSUE TITLE
Theme demo is broken on the Hugo website

### ISSUE THREAD

**Issue by onedrawingperday** [spookey/slick#2]
Hello @spookey 

Recently we had to [make a change](https://github.com/gohugoio/hugoThemes/pull/547) to the Hugo themes site and since then your theme's demo is broken with the following ERROR:

```
 ==== PROCESSING  slick  ====== 
Building site for theme slick using its own exampleSite to ../themeSite/static/theme/slick/
ERROR 2019/02/24 14:43:46 render of "page" failed: execute of template failed: template: _default/single.html:5:7: executing "main" at <partial "post_meta.h...>: error calling partial: "/home/alex/work/src/hugoThemes/slick/layouts/partials/post_meta.html:12:18": execute of template failed: template: partials/post_meta.html:12:18: executing "partials/post_meta.html" at <$.Site.GetPage>: error calling GetPage: failed to resolve page: page reference "/tags/theme" is ambiguous
ERROR 2019/02/24 14:43:46 render of "page" failed: execute of template failed: template: _default/single.html:5:7: executing "main" at <partial "post_meta.h...>: error calling partial: "/home/alex/work/src/hugoThemes/slick/layouts/partials/post_meta.html:12:18": execute of template failed: template: partials/post_meta.html:12:18: executing "partials/post_meta.html" at <$.Site.GetPage>: error calling GetPage: failed to resolve page: page reference "/categories/development" is ambiguous
ERROR 2019/02/24 14:43:46 render of "page" failed: execute of template failed: template: _default/single.html:5:7: executing "main" at <partial "post_meta.h...>: error calling partial: "/home/alex/work/src/hugoThemes/slick/layouts/partials/post_meta.html:12:18": execute of template failed: template: partials/post_meta.html:12:18: executing "partials/post_meta.html" at <$.Site.GetPage>: error calling GetPage: failed to resolve page: page reference "/tags/hugo" is ambiguous
ERROR 2019/02/24 14:43:46 render of "page" failed: execute of template failed: template: _default/single.html:5:7: executing "main" at <partial "post_meta.h...>: error calling partial: "/home/alex/work/src/hugoThemes/slick/layouts/partials/post_meta.html:12:18": execute of template failed: template: partials/post_meta.html:12:18: executing "partials/post_meta.html" at <$.Site.GetPage>: error calling GetPage: failed to resolve page: page reference "/tags/go" is ambiguous
FAILED to create exampleSite for slick
```

There is really no need to use the `GetPage` function for your purposes and it seems that the above ERROR is relevant to what is being discussed [here](https://github.com/gohugoio/hugoDocs/issues/546)

To have your demo generated on the themes showcase once again, you need to change [lines 12-13](https://github.com/spookey/slick/blob/master/layouts/partials/post_meta.html#L12-L13) of `partials/post_meta.html` to the following:

```
      {{ $perma := lower (printf "%s/%s" $taxo (urlize $name)) }}
      <a href="{{ $perma | absURL }}">{{ $name }}</a>
```

And [lines 8-10](https://github.com/spookey/slick/blob/master/layouts/_default/terms.html#L8-L10) of `layouts/_default/terms.html` to the following:

```
      {{ $perma := lower (printf "%s/%s" $type (urlize .Name)) }}
      <li>
      <a href="{{ $perma | absURL }}">{{ .Name }}</a>
```

Note that I have removed the first forward slash `/` from your `printf` function above because a `URL` that begins this way will always point to the host root. Your theme demo will be published in a sub-directory of the Hugo Themes website and as a result a `URL` that begins with a forward slash will throw a 404.

Once you look into the above please let me know.

Thank you

cc: @digitalcraftsman

**Comment by spookey** [comment]
Thank you for your notice - I was unaware that there are problems running that theme below some subfolder..

I've just changed the mentioned lines - please drop me a line if there are still problems.

**Comment by spookey** [comment]
- well awkward - It seems GitHub does not understand the difference between ``fix #2`` and ``should fix #2``..

Did not mean to close it, reopening in case there are still some problems

**Comment by onedrawingperday** [comment]
@spookey 

I confirm that now when I execute the Themes Site Build Script locally your theme's demo generates. 👍 

The change will be reflected on the Hugo Themes website the next time that themes are updated.
